mail-client/bower/bower-0.10.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multiprocessing
+
+DESCRIPTION="A curses terminal client for the Notmuch email system"
+HOMEPAGE="https://github.com/wangp/bower"
+SRC_URI="https://github.com/wangp/bower/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="app-crypt/gpgme:=
+	>=dev-lang/mercury-11.07
+	sys-libs/ncurses:0="
+RDEPEND="app-crypt/gpgme:=
+	net-mail/notmuch
+	sys-apps/coreutils
+	sys-libs/ncurses:0="
+
+src_prepare() {
+	if has_version "sys-libs/ncurses:0[tinfo]" ; then
+		echo "MLLIBS-bower += -ltinfow" >> src/Mercury.params || die
+	fi
+	eapply_user
```

Comment:
drop this and instead run `default` first thing after `src_prepare(){`

Reply:
nothing here tells you to change anything wrt `Mercury.parmas`.

Reply:
Are you saying the "drop this" comment applies only to `eapply_user`?

Reply:
yes, because:
- `default` contains `eapply_user` in >= `EAPI-6`
- I would not bother to requesting it to be run as first thing in `src_prepare`, because without other changes the whole override could be removed.